e2e-tests/tfm/cypress/support/utils/dateFuncs.js

```diff
@@ -12,8 +12,37 @@ const nowPlusMonths = (numberMonths) => {
   return date;
 };
 
+const getCurrentISOSubmissionMonth = () => {
+  const currentDate = new Date();
+  const currentMonth = currentDate.getMonth() + 1;
+  const formattedMonth = currentMonth < 10 ? `0${currentMonth}` : `${currentMonth}`;
+  return `${currentDate.getFullYear()}-${formattedMonth}`;
+};
+
+/**
+ * @param {string} submissionMonth - The submission month as an ISO month stamp
+ * @returns {{ start: { month: number, year, number }, end: { month: number, year, number } }}
+ */
+const getMonthlyReportPeriodFromISOSubmissionMonth = (submissionMonth) => {
+  const [year, month] = submissionMonth.split('-').map((x) => Number(x));
+  const reportPeriodMonth = month === 1 ? 12 : month - 1;
+  const reportPeriodYear = month === 1 ? year - 1 : year;
+  return {
+    start: {
+      month: reportPeriodMonth,
+      year: reportPeriodYear,
+    },
+    end: {
+      month: reportPeriodMonth,
+      year: reportPeriodYear,
+    },
+  };
+};
```

Comment:
Similarly we can simplify this, and avoid hand-rolling the calculation, by taking advantage of the fact that an ISO month stamp ('yyyy-MM') can be used to initialise a Date object.
```
const reportPeriodDate = subMonths(new Date(submissionMonth), 1);
const reportPeriodMonthAndYear = {
  month: reportPeriodDate.getMonth() + 1,
  year: reportPeriodDate.getFullYear(),
};

return { start: reportPeriodMonthAndYear, end: reportPeriodMonthAndYear };
```